src/jvmMain/kotlin/io/beatmaps/controllers/map.kt

```diff
@@ -175,6 +176,18 @@ fun Route.mapController() {
         }
     }
 
+    get<UserController.RedirectName> {
+        transaction {
+            User.select {
+                User.name eq it.username.lowercase()
```

Comment:
Should be using uniqueName not name and probably only find active accounts.
See https://github.com/beatmaps-io/beatsaver-main/blob/master/src/jvmMain/kotlin/io/beatmaps/api/users.kt#L690
`(User.uniqueName eq it.name) and User.active`